profiler/src/main/java/com/splunk/opentelemetry/profiler/JfrActivator.java

```diff
@@ -103,15 +107,23 @@ private void activateJfrAndRunForever(Config config) {
             .executorService(exportExecutorService)
             .build();
     batchingLogsProcessor.start();
+
+    LogDataBuilder logDataBuilder = createLogDataBuilder(config);
```

Comment:
Is it safe to reuse a `LogDataBuilder` this way? It's a mutable object.

Reply:
Ugh, no, it's not...good call, and THANK YOU!